designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/scenariostatus/ScenarioStatusProvider.scala

```diff
@@ -78,6 +80,36 @@ class ScenarioStatusProvider(
     )
   }
 
+  def getActiveScenariosFor(
```

Comment:
This approach doesn't use status prefetching approach which give performance benefits for >1 scenario status fetching. Do we care about that? Take a look at:
```
        prefetchedDeploymentStatuses <- DBIO.from(
          deploymentStatusesProvider.getBulkQueriedDeploymentStatusesForSupportedManagers(
            processingTypesWithMoreThanOneScenario
          )
        )
```
in method above